npm start does not run with babel-register

npm start is inconsistent by omitting -r babel-register electron parameter

package.json should be fixed to:
    "start": "cross-env NODE_ENV=production electron -r babel-register ./app/"

Doesn't seem to be fixed. Similar error still popping up when doing `yarn install`. `npm install` also breaks.
```
> electron-react-boilerplate@0.12.0 postinstall /home/piggyslasher/Sites/linka
> concurrently "npm run flow-typed" "npm run build-dll" "electron-builder install-app-deps" "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json"

[0]
[0] > electron-react-boilerplate@0.12.0 flow-typed /home/piggyslasher/Sites/linka
[0] > rimraf flow-typed/npm && flow-typed install --overwrite || true
[0]
[1]
[1] > electron-react-boilerplate@0.12.0 build-dll /home/piggyslasher/Sites/linka
[1] > cross-env NODE_ENV=development node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.renderer.dev.dll.js --colors
[1]
[2] electron-builder 19.18.1
[2] Installing app dependencies for arch x64 to /home/piggyslasher/Sites/linka/app
[3] node node_modules/fbjs-scripts/node/check-dev-engines.js package.json exited with code 0
[1] /home/piggyslasher/Sites/linka/webpack.config.renderer.dev.dll.js:5
[1] import webpack from 'webpack';
[1] ^^^^^^
[1]
[1] SyntaxError: Unexpected token import
[1]     at createScript (vm.js:74:10)
[1]     at Object.runInThisContext (vm.js:116:10)
[1]     at Module._compile (module.js:533:28)
[1]     at loader (/home/piggyslasher/Sites/linka/node_modules/babel-register/lib/node.js:144:5)
[1]     at Object.require.extensions.(anonymous function) [as .js] (/home/piggyslasher/Sites/linka/node_modules/babel-register/lib/node.js:154:7)
[1]     at Module.load (module.js:503:32)
[1]     at tryModuleLoad (module.js:466:12)
[1]     at Function.Module._load (module.js:458:3)
[1]     at Module.require (module.js:513:17)
[1]     at require (internal/module.js:11:18)
[1] npm
[1]  ERR! code ELIFECYCLE
[1] npm ERR! errno 1
[1] npm ERR! electron-react-boilerplate@0.12.0 build-dll: `cross-env NODE_ENV=development node --trace-warnings -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.renderer.dev.dll.js --colors`
[1] npm ERR! Exit status 1
[1] npm ERR!
[1]
[1] npm ERR! Failed at the electron-react-boilerplate@0.12.0 build-dll script.
[1] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[1]
[1] npm ERR! A complete log of this run can be found in:
[1] npm ERR!     /home/piggyslasher/.npm/_logs/2017-08-01T16_16_32_188Z-debug.log
[1] npm run build-dll exited with code 1
[2] electron-builder install-app-deps exited with code 0
[0] Error: Unable to find a flow project in the current dir or any of it's parent dirs!
[0] Please run this command from within a Flow project.
[0] npm run flow-typed exited with code 0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-react-boilerplate@0.12.0 postinstall: `concurrently "npm run flow-typed" "npm run build-dll" "electron-builder install-app-deps" "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electron-react-boilerplate@0.12.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/piggyslasher/.npm/_logs/2017-08-01T16_16_33_476Z-debug.log
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 - Bumped all dependencies to latest semver
 - Fixed electron-rebuild script
 - Fixed tests scripts to run on all platforms
+- Skip redux logs in console in test ENV
 
 # 0.13.0 (2018.1.6)
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,9 @@
+# 0.13.1 (2018.1.13)
+- Hot Module Reload (HMR) fixes
+- Bumped all dependencies to latest semver
+- Fixed electron-rebuild script
+- Fixed tests scripts to run on all platforms
+
 # 0.13.0 (2018.1.6)
 
 #### Additions

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "electron-rebuild": "electron-rebuild --parallel --force --types prod,dev,optional --module-dir app",
     "flow": "flow",
     "flow-typed": "rimraf flow-typed/npm && flow-typed install --overwrite || true",
-    "lint": "eslint --cache --format=node_modules/eslint-formatter-pretty .",
+    "lint": "cross-env NODE_ENV=development eslint --cache --format=node_modules/eslint-formatter-pretty .",
     "lint-fix": "npm run lint -- --fix",
     "lint-styles": "stylelint app/*.css app/components/*.css --syntax scss",
     "lint-styles-fix": "stylefmt -r app/*.css app/components/*.css",

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "electron-react-boilerplate",
   "productName": "ElectronReact",
-  "version": "0.13.0",
+  "version": "0.13.1",
   "description": "Electron application boilerplate based on React, React Router, Webpack, React Hot Loader for rapid application development",
   "scripts": {
     "build": "concurrently \"npm run build-main\" \"npm run build-renderer\"",

--- a/package.json
+++ b/package.json
@@ -195,7 +195,7 @@
     "history": "^4.7.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "react-hot-loader": "3.1.3",
+    "react-hot-loader": "^4.0.0-beta.13",
     "react-redux": "^5.0.6",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",

